org.texmacs.TeXmacs.yml

```diff
@@ -0,0 +1,116 @@
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: "23.08"
+    directory: texlive
+    autodelete: false
+  org.lyx.LyX.Plugin:
+    versions: stable
+    directory: plugins
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+    add-ld-path: lib
+
+modules:
+
+  - pythonSoft.yml # symPy, numPy
+
+  - name: texliveYextensions
+    buildsystem: simple
+    build-commands:
+      - install -d /app/texlive
+      - install -d /app/plugins
+
+  - name: dictEs-aspell
+    buildsystem: simple
+    build-commands:
+      - ./configure
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts install
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/aspell/dict/es/aspell6-es-1.11-2.tar.bz2
+        sha256: ad367fa1e7069c72eb7ae37e4d39c30a44d32a6aa73cedccbd0d06a69018afcc
+  - name: dictEn-aspell
+    buildsystem: simple
+    build-commands:
+      - ./configure
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts install
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/aspell/dict/en/aspell6-en-2020.12.07-0.tar.bz2
+        sha256: 4c8f734a28a088b88bb6481fcf972d0b2c3dc8da944f7673283ce487eac49fb3
+  - name: magick
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.1-21.tar.gz
+        sha256: 09402e5f17c6575ef9f010bb2e21ae1710f1f3426f115ad4317ee9129c32608e
+  - name: graphviz
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/10.0.1/graphviz-10.0.1.tar.gz
+        sha256: 72e1a575e9c382c4fd2267837962bc4582b3d9ad746e338cd8024bdeca124f32
+  - name: gnuplot
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/gnuplot/5.4.10/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: guile
+    buildsystem: autotools
+    config-opts:
+      - --disable-static 
+      - --disable-error-on-warning
+    build-options:
+      arch:
+        aarch64:
+          config-opts:
+            - --build=aarch64-unknown-linux-gnu
+    sources:
+      - type: archive
+        url: http://ftp.gnu.org/gnu/guile/guile-1.8.8.tar.gz
+        sha256: c3471fed2e72e5b04ad133bbaaf16369e8360283679bcf19800bc1b381024050
+      - type: patch
+        path: guile-1.8.8-configure.patch
+
+  - name: xdg-utils # temporary #fake
+    buildsystem: simple
+    build-commands:
+      - touch /app/bin/xdg-icon-resource /app/bin/xdg-mime /app/bin/xdg-desktop-menu 
+      - touch /app/bin/xdg-desktop-icon /app/bin/xdg-icon-resource
+      - chmod +x /app/bin/xdg-*
```

Comment:
What is this for?

Reply:
Some installation scripts use xdg-utils, but in this case they have no effect, the installation is aborted if they are not found.